dashboard/app/models/level.rb

```diff
@@ -59,8 +62,13 @@ def self.model_name
 
   # https://github.com/rails/rails/issues/3508#issuecomment-29858772
   # Include type in serialization.
+  # also include level_concept_difficulty
   def serializable_hash(options=nil)
-    super.merge 'type' => type
+    super(:include => :level_concept_difficulty).merge 'type' => type
+  end
+
+  def changed?
+    super || (level_concept_difficulty && level_concept_difficulty.changed?)
```

Comment:
same as [above](https://github.com/code-dot-org/code-dot-org/pull/7767/files#r59446989) for this line